dependencies.gradle

```diff
@@ -1,6 +1,6 @@
 ext {
-    supportVersion = '25.3.1'
-    rxJavaVersion = '1.3.2'
+    supportVersion = '26.0.2'
```

Comment:
Why not `27.0.2` (newest one at the time of writing)?

Reply:
Good point, it was added at the point of first commit (3 months ago?). Will update.